### DISCUSSION TITLE
Can magic-trace be used on windows ?

### DISCUSSION THREAD

**Discussion by mjnovice** [janestreet/magic-trace#202] [Q&A] [answered]
I couldn't find any document (maybe I missed something), is there any existing support/plan for support with windows ?

**Comment by cgaebel** [comment, marked as answer]
We have no existing plans to add Windows support. You can read a bit more about why [here](https://github.com/janestreet/magic-trace/wiki/How-could-magic-trace-be-made-to-work-on...#windows).
